<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>coordinaterangeDialog</class>
 <widget class="QDialog" name="coordinaterangeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设置经纬范围</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>南</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="lineEdit_west"/>
   </item>
   <item row="5" column="3">
    <widget class="QLineEdit" name="lineEdit_south"/>
   </item>
   <item row="3" column="6">
    <widget class="QLineEdit" name="lineEdit_east"/>
   </item>
   <item row="1" column="3">
    <widget class="QLineEdit" name="lineEdit_north"/>
   </item>
   <item row="3" column="5">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>东</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>北</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="8">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>西</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>coordinaterangeDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>coordinaterangeDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
